Model/Cron.php

```diff
@@ -1532,47 +1532,45 @@ protected function _authorizePayment()
         if (strcmp($this->_success, 'true') == 0) {
             $this->_order->setData('adyen_notification_event_code_success', 1);
         }
-        $fraudManualReviewStatus = $this->_getFraudManualReviewStatus();
+        $fraudManualReviewStatus = $this->getFraudManualReviewStatus();
         $this->adyenOrderPaymentHelper->createAdyenOrderPayment($this->_order, $this->notification, $this->_isAutoCapture());
         $isTotalAmountAuthorised = $this->_isTotalAmount(
             $this->_order->getPayment()->getEntityId(),
             $this->orderCurrency
         );
 
-        // If manual review is NOT active OR no custom status is set on the Adyen config page
-        if ($this->_fraudManualReview != true || $fraudManualReviewStatus == "") {
-            if ($isTotalAmountAuthorised) {
-                $this->_setPrePaymentAuthorized();
-            } else {
-                $this->_order->addStatusHistoryComment(__(sprintf(
-                    'Partial authorisation w/amount %s %s was processed',
-                    $this->_currency,
-                    $this->_adyenHelper->originalAmount($this->_value, $this->_currency)
-                )), false);
-            }
-        } else {
-            $this->_adyenLogger->addAdyenNotificationCronjob(
-                'Ignore the pre authorized status because the order is ' .
-                'under manual review and use the Manual review status'
-            );
-        }
-
-        if ($isTotalAmountAuthorised) {
+        // IF manual review is NOT active OR no custom status is set on the Adyen config page AND total amount is authorised continue with authorisation
+        // ELSEIF total amount is NOT authorised
+        // ELSE add a comment and set the status to the defined fraudManualReviewStatus
+        if ($this->requireFraudManualReview || $fraudManualReviewStatus == "" && $isTotalAmountAuthorised) {
```

Comment:
Could you use parentheses here so that we can be sure which condition is evaluated first.
```suggestion
        if (($this->requireFraudManualReview || $fraudManualReviewStatus == "") && $isTotalAmountAuthorised) {
```

Reply:
Updated in 88f808b